administrator/language/de-DE/plg_system_jooa11y.ini

```
@@ -0,0 +1,170 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SYSTEM_JOOA11Y="System - Joomla Barriere-Checker"
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT="Inhaltscontainer"
+PLG_SYSTEM_JOOA11Y_FIELD_CHECK_ROOT_DESC="Aria Landmark auf der Seite, die auf Barrierefreiheit geprüft wird. Die Standardeinstellung ist <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE="Bereiche ignorieren"
+PLG_SYSTEM_JOOA11Y_FIELD_CONTAINER_IGNORE_DESC="Bestimmte Bereiche innerhalb des Inhaltscontainers ignorieren. Kommas verwenden, um Klassen oder Elemente zu trennen (z.B. #ignore, .ignore)"
+PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT="Lesbarkeitscontainer"
+PLG_SYSTEM_JOOA11Y_FIELD_READABILITY_ROOT_DESC="Aria Landmark auf der Seite, die auf Lesbarkeit geprüft wird. Die Standardeinstellung ist <strong>main</strong>."
+PLG_SYSTEM_JOOA11Y_FIELD_SHOW_ALWAYS="Immer anzeigen"
+PLG_SYSTEM_JOOA11Y_FIELD_SHOW_ALWAYS_DESC="Lädt den Barriere-Checker auf allen Seiten. Dies ist bei der Entwicklung der Website nützlich, sollte aber nicht aktiviert bleiben, sobald die Website live ist."
+PLG_SYSTEM_JOOA11Y_XML_DESCRIPTION="Der Joomla Barriere-Checker hebt häufige Probleme mit der Zugänglichkeit und Benutzerfreundlichkeit visuell hervor. Das Plugin, das sich an Autoren von Inhalten richtet, identifiziert Fehler oder Unklarheiten und gibt Hinweise, wie diese behoben werden können."
+
+;Global text
+PLG_SYSTEM_JOOA11Y_ALERT_CLOSE="Schließen"
+PLG_SYSTEM_JOOA11Y_ALERT_TEXT="Alarm"
+PLG_SYSTEM_JOOA11Y_CONTAINER_LABEL="Barriere-Checker"
+PLG_SYSTEM_JOOA11Y_CONTRAST="Kontrast"
+PLG_SYSTEM_JOOA11Y_DARK_MODE="Dunkler Modus"
+PLG_SYSTEM_JOOA11Y_ERROR="Fehler"
+PLG_SYSTEM_JOOA11Y_FORM_LABELS="Formular Labels"
+PLG_SYSTEM_JOOA11Y_GOOD="Gut"
+PLG_SYSTEM_JOOA11Y_HIDE_OUTLINE="Gl. ausblenden"
```

```suggestion
PLG_SYSTEM_JOOA11Y_HIDE_OUTLINE="Gliederung ausblenden"
```
Oder ist das zu lang?

---

Das habe ich vermutet. Ich musste erst raten, was Gl. ist

---

Ist zu lang, leider ist dieses kleine Fenster so unflexibel...
Es wird aber klar, sobald man geklickt hat. Da dann oben Gliederung der Seite steht.
Gl. ist die offizielle Abkürzung.